tiatoolbox/cli/visualize.py

```diff
@@ -0,0 +1,93 @@
+"""Command line interface for visualization tool."""
+from __future__ import annotations
+
+import os
+import subprocess
+from pathlib import Path
+from threading import Thread
+
+import click
+import pkg_resources
```

Comment:
Please replace this with, we replaced this in tiatoolbox/__init__.py
if sys.version_info >= (3, 9):  # pragma: no cover
    import importlib.resources as importlib_resources
else:  # pragma: no cover
    import importlib_resources  # To support Python 3.8